extension/services/youtube/youtube.js

```diff
@@ -90,6 +90,15 @@ class YouTube extends serviceList.Service {
       await this.callOneTab(tab.id, "move", { direction });
     }
   }
+
+  playAlbum() {
+    const e = new Error(
+      "Sorry, Youtube playlist or album support is not currently available in Firefox Voice."
+    );
+    e.displayMessage =
+      "Sorry, Youtube playlist or album support is not currently available in Firefox Voice.";
+    throw e;
```

Comment:
I think it would actually be OK to just search for "[query] album" in these cases. E.g., if I want to play the Dark Side Of The Moon album, I can search for [dark side of the moon album](https://www.youtube.com/results?search_query=dark+side+of+the+moon+album) and it gives me a pretty good result.

Reply:
Added support for youtube album.

Reply:
Added support for the album on youtube.